plugins/rogue/scripts/hook.ps1

```
@@ -205,20 +230,34 @@ $baseUrl = $creds['ROGUE_BASE_URL']
+#   NAME:  ROGUE_ACTOR_NAME -> local-part of CLAUDE_CODE_USER_EMAIL
+#          -> git config user.name -> USERNAME -> marker unknown
+# The explicit ROGUE_ACTOR_* values are screened too - compiled bundles already
+# in the field bake a git-config pre-seed into ${CLAUDE_PLUGIN_ROOT}\env, so a
+# plugin update can only fix them if we distrust a poisoned value. See actor.sh.
+$actorName = Select-ActorValue @(
+    $creds['ROGUE_ACTOR_NAME'],
+    (($env:CLAUDE_CODE_USER_EMAIL -split '@')[0])
+)
+if (-not $actorName) {
+    $gitName = ''
+    try { $gitName = (& git config --global user.name 2>$null | Out-String).Trim() } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)